### PR TITLE
Updated PATCH deployment api to new api go changes and added dgraph cloud backend creation

### PR DESCRIPTION
**Pull request by vivetha-madesh**

### CODE REVIEW

File: lib/slash_api_library/SlashAPI/components/handlers/deployments/deployments.py [hunk 1]
```diff
@@ -31,10 +31,22 @@ def create_deployment(session_alias,
...
+                          dgraphHA='false',
+                          size="medium",
+                          alphaStorage='40',
                           deployment_subdomain=None,
                           organization=None,
+                          deploymentMode="graphql",
                           expected_response=200):
-        properties = {"name": deployment_name, "zone": deployment_zone}
+        properties = {"name": deployment_name, "zone": deployment_zone, "deploymentType" : deploymentType}
+        logger.info(deploymentType)
```
Comment (kaushik691) [inline lib/slash_api_library/SlashAPI/components/handlers/deployments/deployments.py:43]: need to check for logging types.. info, debug etc

File: test_suites/slash/slash_api/super_admin.robot [hunk 2]
```diff
@@ -0,0 +1,149 @@
+*** Settings ***
+Documentation     This is a simple test with Robot Framework
```
Comment (kaushik691) [inline test_suites/slash/slash_api/super_admin.robot:2]: documentations missing

File: test_suites/slash/slash_api/super_admin.robot [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+Library           SlashAPI
+Library           Collections
+Variables         ../../../conf/slash/slash_api/variables.py
+
+*** Variables ***
+${Session_alias}    Session1
+
+*** Test Cases ***
+Update Deployment mode
+    Update Deployment    ${Session_alias}    ${URL}    ${HEADERS}    ${deployment_id}    test-edit    deployment_mode=flexible
```
Comment (kaushik691) [inline test_suites/slash/slash_api/super_admin.robot:17]: docs missing

File: lib/slash_api_library/SlashAPI/components/handlers/deployments/deployments.py (hunk 1 above)
Comment (vivetha-madesh) [reply lib/slash_api_library/SlashAPI/components/handlers/deployments/deployments.py]: That was a debug line removed it

**Review by kaushik691 — APPROVED** [review]

**Pull request merged**